webtool/test_diar_eval.py

```
@@ -0,0 +1,174 @@
+    assert not de.EVAL.startswith(str(tmp_path))
+
+
+def test_lauf_name_laesst_keine_pfadanteile_durch():
+    """Der Laufname geht ungeprueft in einen Dateipfad. Ohne diese Wache schriebe
+    `run ../../projekte/x` genau dorthin, wo es verboten ist."""
+    # Der Backslash MUSS doppelt stehen: `"a\b"` ist ein BACKSPACE-Zeichen, und der Test
+    # pruefte damit etwas anderes, als er behauptet — er blieb gruen, weil auch ein Backspace
+    # kein `isalnum()` ist. Sichtbar wird das nur beim Kompilieren (SyntaxWarning „invalid
+    # escape sequence"), nicht am Testergebnis.
+    for schlecht in ("../x", "a/b", "a\\b", ".versteckt", "", "a:b", "..", "x/../y"):
+        with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)